--- phpunit.xml.dist	2018-01-12 09:34:43.000000000 +0100
+++ phpunit.xml.dist	2018-01-12 09:35:06.000000000 +0100
@@ -55,11 +55,7 @@
   <!-- Filter for coverage reports. -->
   <filter>
     <whitelist>
-      <directory>./includes</directory>
-      <directory>./lib</directory>
-      <directory>./modules</directory>
-      <directory>../modules</directory>
-      <directory>../sites</directory>
+      <directory>../modules/guernsey</directory>
       <!-- By definition test classes have no tests. -->
       <exclude>
         <directory suffix="Test.php">./</directory>
